### PR TITLE
Update SIG Multicluster docs

### PR DESCRIPTION
**Pull request by csbell**
Markdown conversion for [SIG Update doc](https://drive.google.com/open?id=1V8NuRqDoZwugf5Tz8K-SFEJbQ4CCiCTZbU8UBzu8euk) distributed and commented on over the past week(s).

/sig-multicluster

### CODE REVIEW

File: sig-multicluster/README.md [hunk 1]
```diff
@@ -24,17 +25,59 @@ A Special Interest Group focussed on solving common challenges related to the  m
...
+
+## Cluster Registry
+Common abstraction for a Registry of Clusters that can store per-Cluster metadata and supports Kubernetes label selection. The Cluster Registry can be built as a standalone or an aggregated API server and currently provides a Registry of Clusters without any actively reconciling Kubernetes controller. The API design is documented here and is intended to serve as a basis to develop multicluster controllers.
+
+|  |  |
+| -------------- |:-------|
+| Recommended for Production | Not yet. |
+| General Availability | Beta by 2018-Q1, GA by 2018-Q2 (see Project plan for details). |
+| Current Level of Activity | Alpha, active development for Beta and GA milestones. |
+| Owner(s) | [@perotinus](https://github.com/perotinus), [@ifont](https://github.com/ifont) |
```
Comment (pmorie) [inline sig-multicluster/README.md]: Ivan's gh nick is actually @font

File: sig-multicluster/README.md [hunk 2]
```diff
@@ -24,17 +25,59 @@ A Special Interest Group focussed on solving common challenges related to the  m
...
+
+|  |  |
+| -------------- |:-------|
+| Recommended for Production | No code yet (discussions only). |
+| General Availability | No. Prototypes only at this point. |
+| Current Level of Activity | Once to twice weekly discussions as part of Federation working group (most active SIG-Multicluster members). |
+| Owner(s) | Working Group |
+| Where to find it? | Code prototypes in development (one example here). Summary of meetings notes and meeting times [available](ttps://docs.google.com/document/d/1v-Kb1pUs3ww_x0MiKtgcyTXCAuZlbVlz4_A9wS3_HXY). |
+
+## Cluster Registry
```
Comment (pmorie) [inline sig-multicluster/README.md]: Note: we'll have to update the name here when the component is renamed, as seems likely.

File: sig-multicluster/README.md [hunk 3]
```diff
@@ -24,17 +25,59 @@ A Special Interest Group focussed on solving common challenges related to the  m
...
+## Federation v2
+Control Plane for newer Multicluster-specific APIs. Discussions are ongoing to focus future development on multi-cluster specific Kubernetes APIs rather than a reimplementation of single-cluster Kubernetes APIs.
+
+|  |  |
+| -------------- |:-------|
+| Recommended for Production | No code yet (discussions only). |
+| General Availability | No. Prototypes only at this point. |
+| Current Level of Activity | Once to twice weekly discussions as part of Federation working group (most active SIG-Multicluster members). |
+| Owner(s) | Working Group |
+| Where to find it? | Code prototypes in development (one example here). Summary of meetings notes and meeting times [available](ttps://docs.google.com/document/d/1v-Kb1pUs3ww_x0MiKtgcyTXCAuZlbVlz4_A9wS3_HXY). |
```
Comment (pmorie) [inline sig-multicluster/README.md]: Looks like `(one example here)` should be a link.

File: sig-multicluster/README.md [hunk 4]
```diff
@@ -24,17 +25,59 @@ A Special Interest Group focussed on solving common challenges related to the  m
...
+| General Availability | CLI: Beta by 2018-Q1, Controller: Alpha by 2018-Q1 |
+| Current Level of Activity? | CLI is alpha and team is looking for feedback. Active development is on to bring it to beta. |
+| Owner(s) | [@nikhiljindal](https://github.com/nikhiljindal), [@G-Harmon](https://github.com/G-Harmon) |
+| Where to find it? | https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress |
+
+
+
+## Federation v1 (a.k.a. Cluster Federation)
+Control Plane that Federates a subset of existing single-cluster Kubernetes API across clusters. Initially released as part of Kubernetes 1.3, Cluster Federation has implemented parts of the existing “single-cluster” Kubernetes API so that they may be used transparently in a multi-cluster fashion. As of Kubernetes 1.9, many of the non-Federated Kubernetes Workloads APIs have moved to GA (Deployments, ReplicaSets, Daemonset, StatefulSet). However, under Federation, these workloads have not reached the maturity of “GA” since they are re-implemented by different federated controllers. The SIG is currently re-thinking how to support Kubernetes APIs in a Federation fashion and this discussion is reflected under Federation v2.
```
Comment (marun) [inline sig-multicluster/README.md]: Consider s/Federates/federates/ and defining what 'federates' means (e.g. coordinate configuration across multiple clusters).

File: sig-multicluster/README.md (hunk 1 above)
Comment (font) [reply sig-multicluster/README.md]: Thanks @pmorie!

**Comment by cblecker** [comment]
@csbell CI is failing as you are editing autogenerated parts of this document (as noted in the comment at the top of the file). If you would like to add the additional subproject detail, there is a custom content marker at the bottom that you can put the additional details in. Additionally, if you'd like to add details of the second meeting, this can be done in sigs.yaml and then regenerating the documents with `make`. Note that regenerating them will delete any changes that are outside of the custom content section.

**Comment by csbell** [comment]
Thanks all. PTAL.

**Pull request merged**